deno.json

```diff
@@ -25,17 +25,15 @@
         "noUncheckedIndexedAccess": false,
         "strictNullChecks": true
     },
-    "imports": {
-        "effect": "./packages/effect/src/index.ts",
-        "effect/": "./packages/effect/src/"
-    },
     "exclude": [
         "**/node_modules/",
         "**/build/",
         "**/dist/",
         "**/bundle/",
         "scratchpad/",
         "scripts/",
-        "vitest.setup.ts"
+        "vitest.setup.ts",
+        "packages/effect/test/Schema.test.ts",
+        "packages/effect/test/SchemaToSerializer.test.ts"
```

Comment:
Unlike `pnpm check`, both tsserver and `deno check` error with this:

```
TS2417 [ERROR]: Class static side 'typeof MyError' incorrectly extends base class static side 'typeof ErrorConstructor'.
  Types of property 'prototype' are incompatible.
    Property 'message' is missing in type 'ErrorConstructor' but required in type 'Error'.
      class MyError extends Error {}
```

Reply:
Still happening, not sure why.

Reply:
🤦‍♂️, forgot I ignored these. Still not sure why it's erroring (some default .d.ts differences methinks), but test typechecks aren't a complete blocker.